src/main/java/steps/AESteps.java

```diff
@@ -25,6 +26,22 @@ public void iFillInInvalidEmail (String word) {
         calcPage.fillInInvalidEmail(word);
     }
 
+    @And("^I fill in with my valid password \"([^\"]*)\"$")
+    public void IFillInValidPassword (String word) {
```

Comment:
Please use unic String. You have been using word 2 times `String word`

Reply:
Found specific variables for the String
`public void iVerifyErrorMessage (String message){
        calcPage.verifyErrorMessage(message); is `

` public void IFillInValidPassword (String password) {
        calcPage.fillInValidPassword(password);`

`public void iFillInInvalidEmail (String email) {
        calcPage.fillInInvalidEmail(email);`